test/test_restore_keeps_panel_hardware.py

```
@@ -0,0 +1,75 @@
+BIG = {"display": {"hardware": {"cols": 128, "rows": 64, "chain_length": 4,
+                                "hardware_mapping": "adafruit-hat-pwm"},
+                   "runtime": {"gpio_slowdown": 4}},
+       "timezone": "America/New_York", "some-plugin": {"enabled": True}}
+SMALL = {"display": {"hardware": {"cols": 64, "rows": 32, "chain_length": 2,
+                                  "hardware_mapping": "regular"},
+                     "runtime": {"gpio_slowdown": 2}},
+         "timezone": "UTC"}
+
+
+def _run(tmp, keep):
+    src = tmp / "backup_config.json"; src.write_text(json.dumps(BIG))
```

Ruff (0.16.1)

[error] Multiple statements on one line (semicolon)

(E702)